Adapters/Vungle/ISVungleAdapter/src/main/java/com/ironsource/adapters/vungle/banner/VungleBannerAdapter.kt

```diff
@@ -218,8 +218,9 @@ class VungleBannerAdapter(adapter: VungleAdapter) :
         try {
             if (bannerSize.isAdaptive && vungleAdSize != null) {
```

Comment:
one more qn should we have to check if the vungleAdSize != null here? just to confirm.

Reply:
Yes. We need it. After removing it, I got compile error.